.gitignore

```diff
@@ -9,6 +9,41 @@ Cargo.lock
 # These are backup files generated by rustfmt
 **/*.rs.bk
 
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
```

Comment:
I think this is unnecessary comment.

Reply:
Changed to what ignores for, who added them.